app/graphql/mutations/create_submission_mutation.rb

```diff
@@ -1,50 +1,41 @@
 # frozen_string_literal: true
 
 module Mutations
-  module CreateSubmissionMutation
-    Definition =
-      GraphQL::Relay::Mutation.define do
-        name 'CreateSubmissionMutation'
+  class CreateSubmissionMutation < Mutations::BaseMutation
+    argument :artist_id, String, required: true
 
-        input_field :additionalInfo, types.String
-        input_field :artistID, !types.String
-        input_field :authenticityCertificate, types.Boolean
-        input_field :category, Types::CategoryType
-        input_field :currency, types.String
-        input_field :depth, types.String
-        input_field :dimensionsMetric, types.String
-        input_field :edition, types.Boolean
-        input_field :editionNumber, types.String
-        input_field :editionSize, types.Int
-        input_field :height, types.String
-        input_field :locationCity, types.String
-        input_field :locationCountry, types.String
-        input_field :locationState, types.String
-        input_field :medium, types.String
-        input_field :minimumPriceDollars, types.Int
-        input_field :provenance, types.String
-        input_field :signature, types.Boolean
-        input_field :state, Types::StateType
-        input_field :title, types.String
-        input_field :width, types.String
-        input_field :year, types.String
+    argument :additional_info, String, required: false
```

Comment:
Do you know if mutations `camelCase` by default? This was an area that had to be explicitly matched between old and new schema, as this is what Eigen uses.

Reply:
This is a good question, I will research!

Reply:
Yes, the default behavior is to camelCase and an option exists to opt out per field.